content/posts/2025-10-28-whats-new-1-10/index.md

```diff
@@ -0,0 +1,203 @@
+
+This article covers only the new features of 1.10.  You can read about the older releases in my other [What's new in Network Observability](https://developers.redhat.com/author/steven-lee) articles.  For a comprehensive coverage of all features, see the official [Red Hat Network Observability documentation](https://docs.redhat.com/en/documentation/openshift_container_platform/4.20/html/network_observability/index).
+
+The starting point is that you have Red Hat OpenShift web console running and the command line `oc` on your computer.
+
+## FlowCollector Setup UI
+
+After [installing Network Observability Operator](https://docs.redhat.com/en/documentation/openshift_container_platform/4.20/html/network_observability/installing-network-observability-operators), you can create a FlowCollector custom resource or instance.  This allows you to configure the many parameters of network observability.  In this release, there is a new 4-step wizard described below that simplifies the creation of this resource.
+
+### Step 1: Overview
+
+This gives an overview of what FlowCollector is and some basic configuration.  The field, **Deploy policies**, will be covered in the section "Improve Security with Network Policies".  If you prefer the traditional Form view or the YAML view, since many parameters are not shown in the wizard, click the **FlowCollector form** link in the second paragraph.
+
+![FlowCollector Setup - Step 1](flowcollector-p1.png)<br>
+Figure 1: FlowCollector Setup - Step 1
+
+### Step 2: Processing
+
+The **Deployment model** is where you can enable Kafka if desired.  This is typically necessary for large clusters.  You can also enable various eBPF features, many of which help with troubleshooting.
+
+![FlowCollector Setup - Step 2](flowcollector-p2.png)<br>
+Figure 2: FlowCollector Setup - Step 2
+
+### Step 3: Loki
+
+To store flow logs, you need Loki. Without it, there will not be a traffic flows table as only metrics will be stored.  The default Loki mode is **LokiStack** instead of **Monolithic**.
+
+![FlowCollector Setup - Step 3](flowcollector-p3.png)<br>
+Figure 3: FlowCollector Setup - Step 3
+
+### Step 4: Consumption
+
+The last step provides guidance on setting the sampling interval.
+
+![FlowCollector Setup - Step 4](flowcollector-p4.png)<br>
+Figure 4: FlowCollector Setup - Step 4
+
+### FlowCollector status
+
+Once you click **Submit** in step 4, it shows a graphical status as FlowCollector initializes and comes up.
+
+![FlowCollector Status](flowcollector-status.png)<br>
+Figure 5: FlowCollector Status
+
+## FlowMetric Setup UI
+
+Like the FlowCollector Setup, there is a new 4-step wizard for FlowMetric Setup.  The FlowMetric CRD lets you define your own custom metrics.  The four steps are **Overview**, **Metric** to define the metric name and type, **Data** to specify the direction and filters, and a **Review** panel before applying the change.  Figure 6 shows what the first step looks like.  The other steps are not shown.
```

Comment:
That feels a bit strange :thinking: 

If you don't want to cover all the screens, let's just remove the step 1 and keep a small sentence here.

Reply:
Or perhaps say something like "we won't go into the details as these steps should be self-explanatory" ?